src/main/kotlin/io/github/seggan/jyxal/runtime/RuntimeMethods.kt

```diff
@@ -379,6 +379,35 @@ fun factors(obj: Any): Any {
     }
 }
 
+fun factorial(stack: ProgramStack): Any {
```

Comment:
The setup is weird, but for monads you simply take an `Any` as input. The element will automagically vectorise